### ISSUE TITLE
Exception

### ISSUE THREAD

**Issue by oschakravarthi** [BabylonJS/BlenderExporter#18]
Traceback (most recent call last):
  File "C:\Users\clrar\AppData\Roaming\Blender Foundation\Blender\2.82\scripts\addons\babylon_js\__init__.py", line 65, in execute
    exporter.execute(context, self.filepath)
  File "C:\Users\clrar\AppData\Roaming\Blender Foundation\Blender\2.82\scripts\addons\babylon_js\json_exporter.py", line 155, in execute
    self.to_json_file()
  File "C:\Users\clrar\AppData\Roaming\Blender Foundation\Blender\2.82\scripts\addons\babylon_js\json_exporter.py", line 217, in to_json_file
    mesh.to_json_file(file_handler)
  File "C:\Users\clrar\AppData\Roaming\Blender Foundation\Blender\2.82\scripts\addons\babylon_js\mesh.py", line 659, in to_json_file
    write_indice_array(file_handler, 'indices', self.indices)
NameError: name 'write_indice_array' is not defined

location: <unknown location>:-1

**Comment by Palmer-JC** [comment]
`write_indice_array` was added for exporter 6.3.0.  I did not do this, but testing with Blender 2.81, on the default scene it executed, & the file worked in the sand box.

I double checked the line numbers of the error stack, and this test had to do through them.  So cannot reproduce.  Am not ready to start using 2.82 at the moment, so see if the problem lay there, but not finding a function seems unlikely.

Maybe, do some more tests, like the default scene, and or provide full export log files.
